### PR TITLE
Minor language corrections/improvements

### PR DESCRIPTION
**Pull request by tomchadwin**
Also some QGIS3-specific changes:

- remove redundant references to GEOS version
- remove references to on-the-fly reprojection

### CODE REVIEW

File: source/docs/user_manual/introduction/qgis_configuration.rst [hunk 1]
```diff
@@ -426,9 +421,8 @@ This tab helps you configure general settings when :ref:`editing vector layer
 **Curve offset tool**
 
 The next 3 options refer to the |offsetCurve| :sup:`Offset Curve` tool in
-:ref:`sec_advanced_edit`. Through the various settings, it is possible to
-influence the shape of the line offset. These options are possible starting
-from GEOS 3.3.
+:ref:`sec_advanced_edit`. These settingls enable you to
```
Comment (DelazJ) [inline source/docs/user_manual/introduction/qgis_configuration.rst]:
```suggestion
:ref:`sec_advanced_edit`. These settings enable you to
```

File: source/docs/user_manual/introduction/qgis_configuration.rst [hunk 2]
```diff
@@ -452,39 +446,35 @@ CRS Settings
 
 **Default CRS for new projects**
 
-* |radioButtonOff| :guilabel:`Don't enable 'on the fly' reprojection`
-* |radioButtonOn| :guilabel:`Automatically enable 'on the fly' reprojection if
-  layers have different CRS`
-* |radioButtonOff| :guilabel:`Enable 'on the fly' reprojection by default`
-* Select a CRS and :guilabel:`Always start new projects with this CRS`
+Allows you to override the default projection used when creating a new project.
```
Comment (DelazJ) [inline source/docs/user_manual/introduction/qgis_configuration.rst]: Is that really an override as it's applied before user notices anything? This is the main place for this property.
```suggestion
Allows you to set the default projection used when creating a new project.
```

File: source/docs/user_manual/introduction/qgis_configuration.rst [hunk 3]
```diff
@@ -528,49 +518,49 @@ Network Tab
...
     useful only in the context of FTP requests.
 
-Excluding some URLs can be added to the text box below the proxy settings (see
+URLs to be requested directly without a proxy can be added to the text box below the proxy settings (see
 Figure_Network_Tab_).
 
 If you need more detailed information about the different proxy settings,
 please refer to the manual of the underlying QT library documentation at
-http://doc.qt.io/qt-4.8/qnetworkproxy.html#ProxyType-enum
+http://doc.qt.io/qt-5/qnetworkproxy.html#ProxyType-enum
```
Comment (DelazJ) [inline source/docs/user_manual/introduction/qgis_configuration.rst]:
```suggestion
https://doc.qt.io/qt-5/qnetworkproxy.html#ProxyType-enum
```

File: source/docs/user_manual/introduction/qgis_configuration.rst [hunk 4]
```diff
@@ -610,40 +600,40 @@ In the properties window for the project under :menuselection:`Project -->
...
 you can set project-specific options.
 
-The project-specific options overwrite their equivalent in the options
+Project-specific options overwrite their equivalent in the options
 described above.
 
 * In the :guilabel:`General` tab, the **general settings** let you:
 
-  * give a title to the project beside the project file path
+  * give a title to the project instead of the project file path
```
Comment (DelazJ) [inline source/docs/user_manual/introduction/qgis_configuration.rst]: This is not a replacement, these are 2 (actually 3 now - with project home) different settings/properties of the project

File: source/docs/user_manual/introduction/qgis_configuration.rst [hunk 5]
```diff
@@ -654,31 +644,30 @@ described above.
...
   should have random colors assigned to them.
-  There is also an additional section where you can define specific colors for the
-  running project. You can find the added colors in the drop down menu of the color dialog
-  window present in each renderer.
-* The tab :guilabel:`OWS Server` allows you to configure your project in order
-  to publish it online. Here you can define information about the QGIS
+  There is an additional section where you can define specific colors for the
+  running project. You can find the colors added in the color dialog's drop-down menu
+  in each renderer.
+* The :guilabel:`OWS Server` tab configures your project in order
```
Comment (DelazJ) [inline source/docs/user_manual/introduction/qgis_configuration.rst]:
```suggestion
* The :guilabel:`QGIS Server` tab configures your project in order
```

**Comment by havatv** [comment]
Thanks again, @tomchadwin !

**Comment by DelazJ** [comment]
1. I have some comments not addressed. Can we give more time to PR before merging please? Few hours for so many lines should not be too much imo.
1. This merge should be reverted as this commit is against 2.18 branch.

**Comment by havatv** [comment]
Sorry about the rush, and not checking the branch.  I have tried to revert.

**Comment by DelazJ** [comment]
@tomchadwin could you try to cherry-pick your changes on a master-derived branch for a new PR, please? Thanks.

**Pull request merged**